app/src/core/document.ts

```diff
@@ -61,7 +73,18 @@ export async function deserializeDocument(
   if (!documentFile) {
     throw Error('document.json missing in audapolis file');
   }
-  const content = JSON.parse(await documentFile.async('text')) as ParagraphGeneric<ParagraphItem>[];
+  const parsed = JSON.parse(await documentFile.async('text')) as DocumentJson;
+  let content;
+  if (!('version' in parsed)) {
+    content = parsed;
+  } else if (parsed.version == 1) {
```

Comment:
Do we already have enough users that we need to support pre-v1-documents? Or would it be okay to just say "Only V1 is supported from now on"